observability/cockpit/reference-content/cockpit-limitations.mdx

```diff
@@ -23,14 +23,36 @@ This page provides information about the capabilities and limits of Scaleway's O
 
 [Logs](/observability/cockpit/concepts/#logs) data is retained for 31 days. After this period, data older than 31 days is deleted.
 
+## Limits for Loki
+
+The following table provides information about [Loki](/observability/cockpit/concepts/#loki)'s limits.
+
+| Parameter                          | Default value | Value for the user | Description                                                                          |
+|------------------------------------|---------------|--------------------|--------------------------------------------------------------------------------------|
+| `ingestion_rate_mb`                | 4             | 4                  | Maximum number of samples accepted in megabytes. The following samples are rejected  |
+| `ingestion_burst_size_mb`          | 6             | 6                  | Ability to exceed `ingestion_rate_mb` for a short period of time (still undefined)   |
```

Comment:
This doesn't seem that clear - particularly the "still undefined" bit? Is it the amount of MB it can exceed the ingestion rate, but the time period is undefined?